src/actions/admin.ts

```diff
@@ -128,6 +128,18 @@ export function AdminActions<ActionsBase extends Constructor<TinlakeParams>>(Bas
         this.contract('FEED').file(web3.fromAscii('maturityDate').padEnd(66, '0'), nftId, timestampSecs)
       )
     }
+
+    getSubmissionPeriod = async () => {
+      return (await this.contract('COORDINATOR').submissionPeriod())
+    }
+
+    getChallengeTime = async () => {
+      return (await this.contract('COORDINATOR').challengeTime()).toBN()
+    }
+
+    getMinimumEpochTime = async () => {
```

Comment:
Ah, I also added this. I think if you pull the `v3` branch, this can be removed.

Reply:
Oh, I added it to `coordinator.ts` though.

Reply:
i can rm it from admin.ts